scss/_utilities.scss

```diff
@@ -615,6 +616,8 @@ $utilities: map-merge(
         1: $border-radius-sm,
         2: $border-radius,
         3: $border-radius-lg,
+        4: $border-radius-xl,
```

Comment:
Check in Bootstrap if `rounded-4` and `rounded-5` weren't defined for examples. Otherwise remove it in Boosted.